beta-src/src/state/game/game-api-slice.ts

```diff
@@ -374,13 +460,21 @@ const gameApiSlice = createSlice({
             },
           };
           setCommand(state, command, "mapCommands", "all");
-          if (currentOrders) {
-            const orderToUpdate = currentOrders.find(
-              (o) => o.unitID === currOrderUnitID,
-            );
-            if (orderToUpdate) {
+
+          if (currentOrders?.length) {
+            if (phase === "Retreats") {
```

Comment:
couldn't this check be made above with a single if check?

`(currentOrders?.length && phase === "Retreats")`

Reply:
That would require two if statements (both checking for current orders length) because that phase check also has an else (if else inside of an if). I just figured either way works so I went with this